releasenotes/notes/add-gates-to-Clifford-class-7de8d3213c60836a.yaml

```diff
@@ -1,10 +1,10 @@
 ---
 fixes:
   - |
-    Add the following Clifford gates, that already exist in the Circuit Library,
+    Add the following Clifford gates, that already exist in the circuit library,
     to the :class:`.Clifford` class:
     :class:`.SXGate`, :class:`.SXdgGate`, :class:`.CYGate`, :class:`.DCXGate`,
     :class:`.iSwapGate` and :class:`.ECRGate`.
   - |
     Add a decomposition of an :class:`.ECRGate` into Clifford gates (up to a global phase)
-    into the :class:`.EquivalenceLibrary`.
```

Comment:
The decomposition is added to the `StandardEquivalenceLibrary` instance, not the class itself.